src/Silverback.Integration.Kafka/Messaging/Configuration/Kafka/ConfluentClientConfigProxies.cs

```diff
@@ -888,6 +899,18 @@ protected ConfluentConsumerConfigProxy(ClientConfig? clientConfig = null)
         {
         }
 
+        /// <summary>
+        ///     Returns a boolean indicating whether group.id is set.
+        /// </summary>
+        public bool IsGroupIdSet
+        {
+            get
+            {
+                _isGroupIdSet ??= GroupId != GroupIdNotSet;
```

Comment:
This can be a single line I think: `get => _isGroupIdSet ??= GroupId != GroupIdNotSet`.

Ideally though, it could be done the other way around, setting this property from the setter of the `GroupId` property. The property is currently auto-generated so you should modify the generator to add a `virtual` and override it in here....or something like that.

Reply:
Not sure I understood what you meant, I just modified the get (if needed we can talk about it later):
https://github.com/BEagle1984/silverback/pull/172/commits/79f770503b5fd550d24587292afcc5406b003e8d#diff-d29091d06f4064f0248abd42f533f0bb93e76d523e96eb8f37c5072c2e8a8f94